Library/Formula/puddletag.rb

```diff
@@ -55,6 +49,9 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    system "sh create_macos_app_bundle.sh --name Puddletag --icon puddletag.png --script #{bin}/puddletag"
```

Comment:
Ah, I didn't realise there was discussion history there with you and Mike. I've asked him to comment.